libs/network/socketserver.c

```diff
@@ -366,7 +402,9 @@ bool socketServerConnectionsIdle(JsNetwork *net) {
       // send data if possible
       JsVar *sendData = jsvObjectGetChild(socket,HTTP_NAME_SEND_DATA,0);
       if (sendData && !jsvIsEmptyString(sendData)) {
-        int sent = socketSendData(net, socket, sckt, &sendData);
+        JsVar *hostInfo = jsvObjectGetChild(connection, HTTP_NAME_HOST_INFO, 0);
```

Comment:
Maybe just check socketType==UDP here to save the minor slowdown for non-UDP connections?